packages/analytics/client/BUILD.bazel

```diff
@@ -7,10 +7,20 @@ PKG_REQUIRE_NAME = "@kbn/analytics-client"
 
 SOURCE_FILES = glob(
   [
-    "src/**/*.ts",
+    "**/*.ts",
   ],
   exclude = [
+    "**/*.config.js",
```

Comment:
Will we need to manually specify which files to exclude from the build for every package or is the list "automagically" updated? If it's a manual process, can we at least add some sort of linting or build error that will throw if files shouldn't be included in the build?

Having to manually update the bazel config takes practice and I'm concerned that as more teams start to migrate their code to the IDM packages structure, we'll have a lot of issues.

Reply:
For now this list was built manually and is highly inspired on the Repo File Naming Convention RFC proposal. As we add more and more features to the tooling being created to support our IDM project, this list and the rest of the BUILD file content should be automatically generated 😃

Reply:
Thats correct. The only exception are mock packages where we need to expose `.mock` files. For those we need to exclude the line with `"**/*.mock.*",`